doc/authors.rst

```diff
@@ -51,91 +29,45 @@
     <p>Olivier Grisel</p>
     </td>
     <td>
-    <a href='https://github.com/jaquesgrobler'><img src='https://avatars3.githubusercontent.com/u/1378870?v=4' class='avatar' /></a> <br />
-    <p>Jaques Grobler</p>
-    </td>
-    <td>
     <a href='https://github.com/yarikoptic'><img src='https://avatars3.githubusercontent.com/u/39889?v=4' class='avatar' /></a> <br />
     <p>Yaroslav Halchenko</p>
     </td>
     <td>
     <a href='https://github.com/NicolasHug'><img src='https://avatars2.githubusercontent.com/u/1190450?v=4' class='avatar' /></a> <br />
     <p>Nicolas Hug</p>
     </td>
-    <td>
-    <a href='https://github.com/adrinjalali'><img src='https://avatars3.githubusercontent.com/u/1663864?v=4' class='avatar' /></a> <br />
-    <p>Adrin Jalali</p>
-    </td>
     </tr>
     <tr>
     <td>
-    <a href='https://github.com/arjoly'><img src='https://avatars0.githubusercontent.com/u/1274722?v=4' class='avatar' /></a> <br />
-    <p>Arnaud Joly</p>
-    </td>
-    <td>
-    <a href='https://github.com/MechCoder'><img src='https://avatars3.githubusercontent.com/u/1867024?v=4' class='avatar' /></a> <br />
-    <p>Manoj Kumar</p>
+    <a href='https://github.com/adrinjalali'><img src='https://avatars3.githubusercontent.com/u/1663864?v=4' class='avatar' /></a> <br />
+    <p>Adrin Jalali</p>
     </td>
     <td>
     <a href='https://github.com/glemaitre'><img src='https://avatars2.githubusercontent.com/u/7454015?v=4' class='avatar' /></a> <br />
     <p>Guillaume Lemaitre</p>
     </td>
     <td>
-    <a href='https://github.com/weilinear'><img src='https://avatars0.githubusercontent.com/u/2232328?v=4' class='avatar' /></a> <br />
-    <p>Wei Li</p>
-    </td>
-    <td>
-    <a href='https://github.com/paolo-losi'><img src='https://avatars1.githubusercontent.com/u/264906?v=4' class='avatar' /></a> <br />
-    <p>Paolo Losi</p>
-    </td>
-    <td>
-    <a href='https://github.com/glouppe'><img src='https://avatars3.githubusercontent.com/u/477771?v=4' class='avatar' /></a> <br />
-    <p>Gilles Louppe</p>
-    </td>
-    <td>
     <a href='https://github.com/jmetzen'><img src='https://avatars1.githubusercontent.com/u/1116263?v=4' class='avatar' /></a> <br />
     <p>Jan Hendrik Metzen</p>
     </td>
-    </tr>
-    <tr>
-    <td>
-    <a href='https://github.com/jarrodmillman'><img src='https://avatars1.githubusercontent.com/u/123428?v=4' class='avatar' /></a> <br />
-    <p>Jarrod Millman</p>
-    </td>
     <td>
     <a href='https://github.com/amueller'><img src='https://avatars3.githubusercontent.com/u/449558?v=4' class='avatar' /></a> <br />
     <p>Andreas Mueller</p>
     </td>
     <td>
-    <a href='https://github.com/vene'><img src='https://avatars0.githubusercontent.com/u/241745?v=4' class='avatar' /></a> <br />
-    <p>Vlad Niculae</p>
```

Comment:
Vlad has made a contribution and some reviews beginning of 2019 I think? Alternatively, if he positively responded to your email, then he is not in the emeritus core dev list above.

Reply:
I don't think I've been emailed about this. In any case, I'm fine with being removed (actually, I just got an email that I have been removed anyway.) I made some small contribs this year but they are not significant.

Reply:
Damn, sorry @vene, That was my fault, I don't think you should have been removed!